packages/ui/src/components/va-select/VaSelect.demo.vue

```diff
@@ -11,6 +11,107 @@
       />
       <p>Value: {{ defaultSingleSelect.value }}</p>
     </VbCard>
+    <VbCard
+      title="Autocomplete"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultSingleSelect.value"
+        class="mb-6"
+        placeholder="Start to write..."
+        :options="defaultSingleSelect.options"
+        autocomplete
+        highlight-matched-text
+      />
+      <p>Value: {{ defaultSingleSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Multiple autocomplete"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultMultiSelect.value"
+        class="mb-6"
+        placeholder="Start to write..."
+        :options="defaultMultiSelect.options"
+        multiple
+        autocomplete
+        highlight-matched-text
+      />
+      <p>Value: {{ defaultMultiSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Multiple autocomplete with max visible options"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultMultiSelect.value"
+        class="mb-6"
+        placeholder="Start to write..."
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/217971802-9e90acb4-202f-48f7-a59a-760f266f060c.png)

For autocomplete - if value is already selected, and I click on autocomplete - I expect search text to be selected like this:

![image](https://user-images.githubusercontent.com/5394573/217971915-216a973e-126c-4ce7-9891-270fc3b46f2c.png)

This would allow me to search for another value.

Instead - only cursor is being placed.